doc/3/controllers/document/index.md

```diff
@@ -0,0 +1,55 @@
+---
+code: true
+type: page
+title: create
+description: document:create
+---
+
+# create
+
+Creates a new document in the provided index and collection.
+
+---
+
+## Arguments
+
+```java
+public CompletableFuture<ConcurrentHashMap<String, Object>> create(
+      final String index,
+      final String collection,
+      final ConcurrentHashMap<String, Object> document,
+      final String _id,
+      final ConcurrentHashMap<String, Object> options) throws NotConnectedException, InternalException
+```
+
+| Arguments          | Type                                         | Description                       |
+| ------------------ | -------------------------------------------- | --------------------------------- |
+| `index`            | <pre>string</pre>                            | Index                             |
+| `collection`       | <pre>string</pre>                            | Collection                        |
+| `content`          | <pre>ConcurrentHashMap<String, Object></pre> | Content of the document to create |
+| `id`               | <pre>string</pre>                            | Document identifier               |
+| `options`          | <pre>ConcurrentHashMap<String, Object></pre> | Optional parameters               |
+
+---
+
+## Options
+
+| Option     | Type        | Description                                                                                                
+| ---------- | ----------- | ---------------------------------------------------------------------------------- |
+| `refresh`  | string      | If set to `wait_for`, Kuzzle will wait for the persistence layer to finish indexing|
```

Comment:
New SDKs have a `waitForRefresh (boolean)` option instead of that "refresh = 'wait_for'" one. This is easier to understand, and we'll probably make that update in Kuzzle's API in the future.

Check the `query` method in this SDK.